app/views/components/table_tdcell.scala.html

```diff
@@ -0,0 +1,27 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.ui._
+@import controllers.routes._
+@import models.ui.Cell
+
+@(cell: Cell)(implicit request: Request[_], messages: Messages, ctx: PageContext)
+<td class=@if(cell.text.isNumeric){"govuk-table__cell govuk-table__cell--numeric"} else{"govuk-table__cell"}>
```

Comment:
Where does govuk-table__cell--numeric come from? It is not used in the design example even though numeric data is used.

Reply:
Les, Ive asked him where he found it, will get back to you